scenarioo-client/app/useCase/usecase.controller.ts

```diff
@@ -20,19 +20,20 @@ import {ConfigurationService} from '../services/configuration.service';
 import {UseCaseDiffInfoService} from '../diffViewer/services/use-case-diff-info.service';
```

Comment:
UseCaseController is no longer needed and can be deleted. The following steps are necessary:
- In app.routes.ts remove
  ```
  controller: 'UseCaseController', 
  controllerAs: 'useCase',
  ```
- In app.ts remove the import
- Use the test cases in usecase.controller.spec.ts as a guideline to write tests for scenarios-overview.component.ts. Afterwards, delete usecase.controller.spec.ts. For more information see here: https://angular.io/guide/testing#component-test-basics